test/elf/loongarch64_relax-got.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+. $(dirname $0)/common.inc
+
+cat <<'EOF' | $CC -o $t/a.o -c -xassembler -
+.globl get_sym1, get_sym2, get_sym3, get_sym4, get_sym5
+get_sym1:
+  la.global $a0, sym1
+  ld.w $a0, $a0, 0
+  ret
+get_sym2:
+  la.global $a0, sym2
+  ld.w $a0, $a0, 0
+  ret
+get_sym3:
+  la.global $a0, sym3
+  ld.w $a0, $a0, 0
+  ret
+get_sym4:
+  la.global $a0, sym4
+  ld.w $a0, $a0, 0
+  ret
+get_sym5:
+  la.global $a0, sym5
+  ld.w $a0, $a0, 0
+  ret
```

Comment:
I don't think you need five symbols for this test, as they are testing exactly the same relaxation.